hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBCheckpointUtils.java

```diff
@@ -67,4 +68,15 @@ public static boolean waitForCheckpointDirectoryExist(File file)
       return false;
     }
   }
+
+  /**
+   * Wait for checkpoint directory to be created for 5 secs with 100 millis
```

Comment:
update comments. Default value a minute.

Reply:
changed the default value to 20 secs, 60 secs seems too long.